test/test_quantized.py

```diff
@@ -1177,18 +1175,15 @@ def test_qconv_unpack(self, X, strideH, strideW, padH, padW, channelwise):
         qconv_prepack = torch.ops.quantized.conv_prepack
         qconv_unpack = torch.ops.quantized.conv_unpack
 
-        # Orig tensor is assumed to be in K(C/G)RS format
         W = torch.from_numpy(filters).to(torch.float)
-        # K(C/G)RS -> KRS(C/G)
-        W_KRSC = W.permute([0, 2, 3, 1]).contiguous()
         if channelwise:
-            W_q = torch.quantize_linear_per_channel(W_KRSC,
+            W_q = torch.quantize_linear_per_channel(W,
```

Comment:
Thanks for changing this too. Did you test it locally by setting `PYTORCH_TEST_WITH_QNNPACK` ?

Reply:
Yes, I did :)